cli/tools/lint/linter.rs

```diff
@@ -84,10 +98,24 @@ impl CliLinter {
   pub fn lint_with_ast(
     &self,
     parsed_source: &ParsedSource,
-  ) -> Vec<LintDiagnostic> {
-    self
-      .linter
-      .lint_with_ast(parsed_source, self.deno_lint_config.clone())
+    token: CancellationToken,
+  ) -> Result<Vec<LintDiagnostic>, AnyError> {
+    // TODO(bartlomieju): surface error is running plugin fails
```

Comment:
```suggestion
    // TODO(bartlomieju): surface error if running plugin fails
```

I think.

Reply:
That's a leftover, it's actually already addressed :)